### PR TITLE
Remove document_bundle from drop off model

### PR DESCRIPTION
**Pull request by alaroia-cfa**
While working on [Manually add client](https://www.pivotaltracker.com/story/show/175466221), @jenny-heath and I noticed that we don't need `document_bundle` on drop-offs anymore, since we'll be creating `Document` records. This removes the property and related validation & tests.

### CODE REVIEW

**Comment by alaroia-cfa** [comment]
Note that I did check with Kelly M. to validate that we don't need to access this data in the future.

The story has some copy-pasta to delete these records from demo (and production). It struck me as not essential to do that in a data migration.

File: app/models/intake_site_drop_off.rb [hunk 1]
```diff
@@ -106,18 +106,11 @@ class IntakeSiteDropOff < ApplicationRecord
   validates :certification_level, allow_blank: true, inclusion: { in: CERTIFICATION_LEVELS }
   validates :email, allow_blank: true, 'valid_email_2/email': true
   validates :phone_number, allow_blank: true, phone: true
-  validate :has_document_bundle?
   validate :has_valid_pickup_date?
 
   has_one_attached :document_bundle
```
Comment (bengolder) [inline app/models/intake_site_drop_off.rb:111]: it looks like we should also remove this line in order to remove the property.

- Ben & Sarah

Comment (alaroia-cfa) [reply app/models/intake_site_drop_off.rb:111]: D'oh, yes, I meant to do that. Doing so right now.

Comment (sniemeyer13) [reply app/models/intake_site_drop_off.rb:111]: We've got it covered!

**Pull request merged**